docs/getting-started/users-page.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[users-page]]
+= Users
+
+The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:
+
+[role="screenshot"]
+image::images/users/users-page.png[]
+
+*User Key Performance Indicators (KPIs) charts*
+
+KPI charts show the total number of users, successful and failed user authentications, and unique IPs based on the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.
+
+TIP: Place your cursor inside a KPI chart to view the actions menu (*...*), where you can perform these actions: inspect, open in Lens, add to a new case, and attach to an existing case.
+[role="screenshot"]
+image::images/users/chart-menu.png[]
+
+*Data tables*
+
+Beneath the KPI charts are data tables, which are useful for viewing and investigating specific types of data. Select the relevant tab to view the following details:
+
+* *All users*: A chronological list of unique user names, the time they were last active, and the associated domain.
+* *Authentications*: A chronological list of user authentication events and their associated details, such as the number of successes, failures, and the hostname of the last successful destination.
+* *Anomalies*: Unusual activity discovered by machine learning jobs that contain user data.
+* *Events*: Ingested events that contain the `user.name` field. You can stack by `event.action`, `event.dataset`, or `event.module` fields.
```

Comment:
I could be wrong but I think it's "field" not "fields" since you can only stack by one OR the other. i.e. "You can stack by the 'event.action' field".